<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des reclamations</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="title">
     <string>Recherche</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Id:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rech_type">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rech_id">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rech_type">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rech_porc">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>1271</width>
      <height>661</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/imageqt.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>571</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 82), stop:1 rgba(255, 255, 255, 255));
border-radius: 25px;


</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small Semibold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Titre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small Semibold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small Semibold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(160, 214, 182);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(160, 214, 182);</string>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(160, 214, 182);</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_afficher">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>390</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(160, 214, 182);</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small Semibold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string> Id reclamation : </string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>270</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>tri</string>
     </property>
     <widget class="QPushButton" name="pushButton_triid">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>tri id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tritype">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>tri titre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_trip">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>tri status</string>
      </property>
     </widget>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit_6</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_modifer</zorder>
    <zorder>pushButton_supprimer</zorder>
    <zorder>pushButton_afficher</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit_5</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>641</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 82), stop:1 rgba(255, 255, 255, 255));
border-radius: 25px;

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>601</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stat">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>450</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Statistiques</string>
     </property>
    </widget>
   </widget>
   <zorder>label_7</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
